skimage/util/montage.py

```diff
@@ -28,21 +28,24 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
 
     Parameters
     ----------
-    arr_in: ndarray, shape=[n_images, height, width]
+    arr_in : ndarray, shape=[n_images, height, width]
         3-dimensional input array representing an ensemble of n_images
         of equal shape (i.e. [height, width]).
-    fill: float or 'mean', optional
+    fill : float or 'mean', optional
         How to fill the 2-dimensional output array when sqrt(n_images)
         is not an integer. If 'mean' is chosen, then fill = arr_in.mean().
-    rescale_intensity: bool, optional
+    rescale_intensity : bool, optional
         Whether to rescale the intensity of each image to [0, 1].
-    grid_shape: tuple, optional
+    grid_shape : tuple, optional
         The desired grid shape for the montage (tiles_y, tiles_x).
         The default aspect ratio is square.
+    border_padding : int, optional
+        The size of the spacing between the tiles to make the 
```

Comment:
This is not exactly true, as the spacing between the tiles will be twice as much.